drivers/jsch/pom.xml

```diff
@@ -90,12 +90,12 @@
     <dependency>
       <groupId>com.jcraft</groupId>
       <artifactId>jsch.agentproxy.jsch</artifactId>
-      <version>0.0.8</version>
+      <version>0.0.9</version>
     </dependency>
     <dependency>
       <groupId>com.jcraft</groupId>
       <artifactId>jsch.agentproxy.connector-factory</artifactId>
-      <version>0.0.8</version>
+      <version>0.0.9</version>
```

Comment:
> I think one or the other is required.

Interesting...it doesn't _seem_ as though `connector-factory` should [depend on `sshj`](https://github.com/ymnk/jsch-agent-proxy/blob/master/jsch-agent-proxy-connector-factory/pom.xml#L16-L42) in any way. Wonder what's going on there...